### DISCUSSION TITLE
Remove gender from category name

### DISCUSSION THREAD

**Discussion by endwaa** [jflamy/owlcms4#885] [General]
I have uploaded a `AgeGroups`-sheet with custom category names. In the "Category"-section the gender is displayed between the category name and the weight class. E.g. a category named "SM" with the weight class 81 is displayed as following: "SM M 81". Is it possible to avoid this behaviour?

Also when I download the results a gender suffix is added, like "M" or "F" after the custom category name.

**Comment by jflamy** [comment]
Currently the category codes are gender-neutral, and so I add the gender.  The exception is Masters where the codes already have the gender in them.  Maybe you can try setting the type as "MASTERS" and see if that works as a quick workaround.

**Reply by jflamy** [reply]
The only side-effect to setting Masters would be changing the start order if you have multiple age groups in a session (it will be older athletes first).

**Comment by jflamy** [comment]
I am currenlty messing with the AgeGroups logic so this is a good time to think about this.  I am currently adding support for naming the Championship.  So one avenue is "if the Championship is named, assume that the categories are named with the gender in them".  This would preserve backwards compatibility.  In the future I would publish the AgeGroups files with championship names and gendered categories.

(am thinking aloud)

**Reply by jflamy** [reply]
Naming the championship is required for example when there are "PanAmerican Masters" at the same time and in the same venue as "Central and SouthAmerican Masters".  Canadians can be in the first one, but not in the second -- so different medals, eligibility categories, maybe event qualifying criteria.   Naming the championship will allow simpler production of the final package with medal counts and team scores.

**Comment by jflamy** [comment]
I have started issue #886 so I can add it to the project board.
In version 48 (currently being worked on) you will be able to add a ! before an age group code so that owlcms will not add the gender to the categories (it will presume that the age group is already gendered).

**Comment by endwaa** [comment]
Excellent, thank you for being so quick to respond 😊
